TombEngine/Game/effects/lightning.h

```diff
@@ -20,6 +20,23 @@ namespace TEN::Effects::Lightning
 		LI_SPARKEND = 16
 	};
 
+	struct TWOGUNINFO
+	{
+		Pose pos;
+		Vector3i pos1;
+		Vector3i pos4;
+		short life;
+		short coil;
+		short spin, spinadd;
+		short length, dlength;
+		byte segments = 56;
```

Comment:
Please demagic `56`, and also we don't use `byte` type, please use `char` or `unsigned char` here.

Reply:
fixed